blockchain/chain.go

```diff
@@ -1318,206 +1411,82 @@ func (b *BlockChain) flushBlockIndexWarnOnly() {
 	}
 }
 
-// connectBestChain handles connecting the passed block to the chain while
-// respecting proper chain selection according to the chain with the most
-// proof of work.  In the typical case, the new block simply extends the main
-// chain.  However, it may also be extending (or creating) a side chain (fork)
-// which may or may not end up becoming the main chain depending on which fork
-// cumulatively has the most proof of work.  It returns the resulting fork
-// length, that is to say the number of blocks to the fork point from the main
-// chain, which will be zero if the block ends up on the main chain (either
-// due to extending the main chain or causing a reorganization to become the
-// main chain).
-//
-// The flags modify the behavior of this function as follows:
-//  - BFFastAdd: Avoids several expensive transaction validation operations.
-//    This is useful when using checkpoints.
+// isOldTimestamp returns whether the given node has a timestamp is too far in
```

Comment:
s / ...has a timestamp is too far... / ...has a timestamp too far ...

Reply:
Corrected.